Jonathan Ho Assignment One

jonathanhoNumberXIII opened the pull request:
First Week Homework (First Assignment for Software General). Enjoy!

jonathanhoNumberXIII committed:
Add Hello World File

jonathanhoNumberXIII committed:
I used the nests to store all of the data separately for each team, then
I printed it after placing their team number into the input variable

jonathanhoNumberXIII committed:
I placed the data of each robotics team into nests of dictionaries and
printed them out through the input variable.

DavidCarlip commented on ch_1_assign_Jonathan_Ho.py:
```diff
@@ -0,0 +1,153 @@
+Robot_Team = input("What Robotics Team do you want to look into?")
```
You need to ask for what piece of information from the team the user is looking for as well as which team

jonathanhoNumberXIII replied:
Whoops! I heard from another Citrus Member that all you needed was the team info as a whole, not as an individual. I'll see to it this is fixed

jonathanhoNumberXIII committed:
I changed the Dictionary code so it's now a nested Dictionary that
contains all the data for each team

jonathanhoNumberXIII committed:
I simply deleted the larger portion of the code, and printed out a
nested dictionary with the input questions inside of it.

jonathanhoNumberXIII committed:
Update ch_1_assign_Jonathan_Ho.py

jonathanhoNumberXIII committed:
My code can add any team, by name or number, and remove them as well as
search to ensure the team is in the database. They can also see certain team information and modify it to their li and modify it to their liking.

jonathanhoNumberXIII committed:
Merge branch 'master' of https://github.com/jonathanhoNumberXIII/software-training

jonathanhoNumberXIII committed:
This assignment is a combination of the two before. I made a nested
dictionary from assignment one and used functions from assignment two to
make it a database with all the dictionaries.

jonathanhoNumberXIII committed:
This modified code allows the user to add, remove, view, modify, search,
and see all the teams within the team database made with several other
teams plopped into it.

jonathanhoNumberXIII committed:
In this bundle of code, I'm using the Pythagorean Theorem for
two-dimensional and three-dimensional shapes, mainly rectangular prisms
and 2D triangles. The user is in a while loop while doing this, and he
or she can exit when they type "exit". Unless they don't want to, they
can enter their own numbers and allow the code to use the Pythagorean
Theorem to calculate the distance between one point and another in the
dimensional shape they choose.

jonathanhoNumberXIII committed:
This code has been updated and modified to make sure that there are no
error messages while maintaining its primary function. This is an
updated version of Chapter 2's assignmet

jonathanhoNumberXIII committed:
Chapter 2 Assignment (Modified Again) This time, this code has only a
blank dictionary, with the add function modified so it can move teams
and data that the user inputs into the blank database. All other
functions have been updated to.

jonathanhoNumberXIII committed:
Update 2.1.0 of Chapter 2

Added additional validations to the add function in case an input wasn't
a number. Also made sure that the user typed yes or no in cam_question,
or it resulted in them having to try again

jonathanhoNumberXIII committed:
Update 2.1.1 of Chapter 2

Modified the remove function so it actually removes the element
specified. The previous code had the remove function remove from a list,
so I changed it to remove it from a dictionary

jonathanhoNumberXIII committed:
Updated Chapter 4 Assignment 2.1.0

Chapter 4 assignment which now has the additional code that calculates
whether the 2D or the 3D point is furthest from the origin, then prints
out the farthest point.

jonathanhoNumberXIII committed:
Chapter 4 Version 2.1.1 (Updated)

Fixed the syntax error. You can now check if the code works or not.

jonathanhoNumberXIII committed:
Update Chapter 2 Assignment Version 2.1.2

Removed the isalpha() method after figuring that the code can work
without it. The use of it actually affects the code in a negative way.
Also made the validations for the inputs in the add function.
